ironmon_tracker/screens/ExtrasScreen.lua

```diff
@@ -171,5 +231,39 @@ function ExtrasScreen.drawScreen()
 	end
 
 	local ivBtn = ExtrasScreen.Buttons.EstimateIVs
-	Drawing.drawText(topboxX + 4, ivBtn.box[2] + ivBtn.box[4] + 1, ivBtn.ivText, Theme.COLORS[ivBtn.textColor], shadowcolor)
-end
\ No newline at end of file
+	if ivBtn then
+		Drawing.drawText(topboxX + 4, ivBtn.box[2] + ivBtn.box[4] + 1, ivBtn.ivText, Theme.COLORS[ivBtn.textColor],
+		shadowcolor)
+	end
```

Comment:
Out of curiosity, why is the if-condition needed here? What changed made this required.

Reply:
This was previously needed before I moved the creation of the log view buttons to their own function, something about the timing of the onClick functions messed up and crashed when clicking/drawing any of the buttons but the log ones. This is not needed anymore though so I'll remove it